tests/unit/amo/components/TestCategory.js

```diff
@@ -137,22 +139,24 @@ describe(__filename, () => {
     expect(root.find(ErrorList)).toHaveLength(1);
   });
 
-  it('fetches categories and landing data when not yet loaded', () => {
+  it('fetches categories and landing data ' +
```

Comment:
You could have use a template literal here - also there's helper tag called `oneLine` that will remove leading whitespace from the second line. See https://github.com/mozilla/addons-frontend/blob/350227964acd5818bacefb5057108678a463a36e/src/core/server/sriDataPlugin.js#L5 for an example.